terraformprovider/resource_object_group.go

```diff
@@ -39,7 +39,7 @@ func resourceObjectGroup() *schema.Resource {
 			},
 			"filter_json": {
 				Type:         schema.TypeString,
-				Default:      `[{"field":"key","regex":".*"}]`,
+				Default:      `{"AND":[{"field":"key","regex":".*"}]}`,
```

Comment:
This is taken from the terraform-modules https://stash.int.klarna.net/projects/KLOG/repos/terraform-modules/browse/ingestion-chaossearch-app/ingestion-chaossearch.tf#51

I changed it because the old option fails on json.Unmarshal - it's an array, not an interface{}